index.js

```diff
@@ -129,10 +129,27 @@ const msg = '<p class="error-msg"> Please write your email in lower case <p> ';
 const regex = /^[a-z0-9+_.-]+@[a-z0-9.-]+$/;
 form1.addEventListener('submit', (e) => {
   const { value } = email;
+  const name = document.getElementById('name').value;
+  const description = document.getElementById('description').value;
   if (!(value.match(regex))) {
     errorMsg.innerHTML = msg;
     e.preventDefault();
+  } else {
+    // Saving Data
+    const contactData = { user_name: name, user_email: value, user_description: description };
+    const storeData = JSON.stringify(contactData);
+    localStorage.setItem('contactData', storeData);
```

Comment:
- [ ] You did great work, but kindly use the `change` event instead of the `submit` event to store in the local storage, so whenever the user adds any data at any input the data get saved in the local storage. As required by the requirements `When the user changes the content of any input field, the data is saved to the local storage.`